x-pack/platform/packages/shared/onechat/onechat-common/chat/conversation.ts

```diff
@@ -108,6 +108,8 @@ export interface ConversationRound {
   steps: ConversationRoundStep[];
   /** The final response from the assistant */
   response: AssistantResponse;
+  /** when tracing is enabled, contains the traceId associated with this round */
+  trace_id?: string;
```

Comment:
I thought about it a bit, and in the end I decided it made sense to log the trace_id directly in the conversation round, so that we can easily retrieve it later. This gets added only when inference tracing is enabled, otherwise the field is not populated.